Sources/Sentry/SentryNSDataTracker.m

```diff
@@ -173,9 +179,32 @@ - (NSData *)measureNSDataFromURL:(NSURL *)url
 
     [ioSpan setDataValue:path forKey:@"file.path"];
 
+    [self mainThreadExtraInfo:ioSpan];
+
     return ioSpan;
 }
 
+- (void)mainThreadExtraInfo:(id<SentrySpan>)span
+{
+    BOOL isMainThread = [NSThread isMainThread];
+
+    [span setDataValue:@(isMainThread) forKey:@"blocked_main_thread"];
```

Comment:
`m`: I guess you already synced `blocked_main_thread` and `call_stack` with other teams and backend?

Reply:
This is from the RFC